requirements.txt

```diff
@@ -6,13 +6,12 @@
 #
 --no-binary psycopg2
 
-git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
-git+https://github.com/Brown-University-Library/django-shibboleth-remoteuser.git#egg=django-shibboleth-remoteuser
-git+https://github.com/mitodl/mit-moira.git#egg=mit-moira
-git+https://github.com/mitodl/pycaption.git#egg=pycaption
+-e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
```

Comment:
The `-e` flags need to be manually removed because they cause issues with production deployment (#447)

Reply:
Ah good catch